docs/visual-basic/programming-guide/concepts/linq/how-to-split-a-file-into-many-files-by-using-groups-linq.md

```diff
@@ -62,7 +62,7 @@ Class SplitWithGroups
         ' Note that nested foreach loops are required to access  
         ' individual items with each group.  
         For Each gGroup In groupQuery  
-            Dim fileName As String = "..'..'..'testFile_" & gGroup.groupKey & ".txt"  
+            Dim fileName As String = "../../../testFile_" & gGroup.groupKey & ".txt"  
             Dim sw As New System.IO.StreamWriter(fileName)  
             Console.WriteLine(gGroup.groupKey)  
             For Each item In gGroup.groupName  
```

Comment:
The "Compiling the Code" section assumes .NET Framework 3.5. It should be changed to reflect multiple versions:

> Create and compile a project that targets one of the following options:
> - .NET Framework version 3.5 with a reference to System.Core.dll.
> - .NET Framework version 4.0 or higher.
> - .NET Core version 1.0 or higher.

The dependence on Visual Studio needs to be made clear as far as file paths are concerned so that developers know to change the path if using CLI tools (or the command line).

Finally, since the console template in VS creates a module and this example creates a class, the project has to be configured to start using Sub Main. It would be best to change the Class...End Class to Module...End Module.